stac_fastapi/pgstac/app.py

```diff
@@ -67,44 +73,68 @@
     "pagination": OffsetPaginationExtension(),
 }
 
+# item_collection extensions
+itm_col_extensions_map = {
+    "filter": FilterExtension(client=FiltersClient()),
+    "pagination": TokenPaginationExtension(),
+}
```

Comment:
At first I tried to removed the duplication of the class init (e.g we do twice `FilterExtension(client=FiltersClient())`), but I couldn't find a nice a clear way